Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -434,6 +450,7 @@ open class ComposerVC: _ViewController,
         }
 
         content.clear()
+        checkChannelCountdown()
```

Comment:
We shouldn't need to call this function, I think the channel countdown logic should be called whenever the content changes, so when `updateContent()` is called. Calling `checkChannelCountdown()` inside the `updateContent()` should be enough right? 🤔 
```suggestion
        checkChannelCountdown()
```

Reply:
Done